### DISCUSSION TITLE
Suggestion: Mobile Clients Page

### DISCUSSION THREAD

**Discussion by mrahmadt** [firefly-iii/firefly-iii#8830] [General]
Hello

Just a suggestion, I tried to search github and the documentation and I didn't see anyone asking the same

It would be very nice if we have a page (maybe in the documentation) , that list firefly-iii mobile clients for Android and iPhone.

**Comment by digitlength** [comment]
https://docs.firefly-iii.org/references/firefly-iii/third-parties/apps/
